docs/deployAutomationForFirewallCostOptimization.md

```diff
@@ -0,0 +1,427 @@
+
+Click create when finished.  
+
+#### Create Automation PowerShell Runbook
+
+<img src="./assets/automation/createAutmationRunbook1Portal.png" alt="Create a User-Assigned Managed Identity for the Automation" />
+
+Navigate to the automation account created, 'aa-ngsa-asb-firewall-automation-dev' and select the 'Runbook' blade, and "Create a runbook.  
+
+<img src="./assets/automation/createAutmationRunbook2Portal.png" alt="Create a User-Assigned Managed Identity for the Automation" />
+
+Create an Azure Powershell Runbook from the Azure Portal within the Automation Resource Group or using the Azure CLI
+
+Use the portal UX upload the Powershell runbook file located in the scripts/automation directory
+
+#### Create Automation Schedules
+
+<img src="./assets/automation/createAutmationAccountSchedule1Portal.png" alt="Create a User-Assigned Managed Identity for the Automation" />
+
+Select the "Schedules" blade on the left side of the Autoamtion Account screen.
+
+<img src="./assets/automation/createRunbookSchedule1Portal.png" alt="Create a User-Assigned Managed Identity for the Automation" />
+
+Establish the schedule desired for the firewall automation.  The Schedule decides for this implementation was to turn the development firewall off at 10:00 PM EST and back on at 6:00 AM EST.  
+
+<img src="./assets/automation/editRunbookScheduleLinked1Portal.png" alt="Create a User-Assigned Managed Identity for the Automation" />
+
+Link the runbook and the schedule
+
+<img src="./assets/automation/endAutomationScheduleLinked1Portal.png" alt="Create a User-Assigned Managed Identity for the Automation" />
+
+Once linked, the defaulted values located within the Runbook will be visible.  
+
+### Azure CLI & Az PowerShell Modules
+
+#### Create the Resource Group with the Azure CLI
+
+The following commands authenticate a user to a specified tenant and then switches into the desired subscription.  
+
+Once authentication and authorization is established the Azure CLI is used to create a resource group.  
+
+   ```bash
+
+    # Log into the tenant
+    az login --tenant {tenant Id}
+
+    # Set the subscription desired    
+    az account set --subscription {subscription name}
+    
+    # pass in resource group  name and location
+    az group create -name {automation resource group name} -location {location}
+
+   ```
+
+#### Create User-Assigned Managed Identity with Azure CLI and PowerShell
+
+Creates the Azure user-assigned managed identity and adds the permissions needed.
+
+``` bash
+
+# Log into the tenant
+az login --tenant {tenant Id}
+
+# Set the subscription desired    
+az account set --subscription {subscription name}
+
+az identity create --resource-group {automation resource group name} --name {user-assigned managed identity name}
+
+# The Az CLI commands for this errored at the time of writing so pwsh was used.
```

Comment:
If there is no task for this yet, I recommend creating a new one error so we can investigate further if needed.